docs/tutorial.md

```diff
@@ -0,0 +1,372 @@
+---
+title: Quarkus Operator Tutorial
+linkTitle: Tutorial
+weight: 30
+description: An in-depth walkthough of building and running a Quarkus-based operator.
+---
+
+**NOTE:** If your project was created with an `operator-sdk` version prior to `v1.0.0`
+please [migrate][migration-guide], or consult the [legacy docs][legacy-quickstart-doc].
```

Comment:
Do these link anywhere? This is not relevant to this java project because there was no way to do this in older versions of the SDK.
```suggestion
```

Reply:
Removed note entirely as prior to `v1.0.0`, we don't have any changes for `java-plugin` in operator-sdk.